app/components/ui/molecules/StaffGrid.js

```diff
@@ -4,7 +4,7 @@ import { Flex, Box } from 'grid-styled'
 const StaffGrid = ({ children, ...props }) => (
   <Flex flexWrap="wrap" mb={-3} mx={-2} {...props}>
     {children.map((item, index) => (
-      <Box key={item.key || index} mb={3} px={2} width={[1 / 2, 1 / 2, 1 / 3]}>
+      <Box key={item.key || index} mb={3} px={2} width={[1 / 1, 1 / 2, 1 / 3]}>
```

Comment:
This can just be `1` rather than `1 / 1`? Ever so slightly easier to read.

@hdrury1 @chugginselifesciences, just to clarify, what is meant by "at the smallest breakpoint" in issue #921?

This is what we have right now:
- one column < 480
- 480 <= two columns < 768

Is that right?

Reply:
@jsms90 Yes.